Project.toml

```diff
@@ -4,11 +4,13 @@ authors = ["Alireza Yazdi <alireza@anyonsys.com>", "Paul Le Roux <pleroux@anyons
 version = "0.1.0"
 
 [deps]
-Coverage = "a2441757-f6aa-5fb2-8edb-039e3f45d037"
```

Comment:
Coverage is required to run coverage.jl

Reply:
I think we can get the best of both if we move the `Coverage` dependency to `test/Project.toml`. Snowflake then has no unneeded dependencies, and we can run the `coverage.jl` script from the test project.